ptx/sec_newton.ptx

```diff
@@ -68,7 +78,13 @@
       <figure xml:id="fig_newt1b">
         <caption/>
         <image xml:id="img_newt1b">
-          <description></description>
+          <shortdescription> A tangent line is drawn to the graph at (x_1,f(x_1)) and and meets on the x-axis at point x_2
+          </shortdescription>
+          <description>
+            <p>the second graph has values (1,0.5, 0, -0.5, -1) on the y axis,while the X-axis represents iterations <m></m>
```

Comment:
Capitalize 'the'.

Put math tags around the list of values, and the x, y variable names. (Also, use lower case.)